tools/ansible/tasks/wsu/test/e2e/wsu_test.go

```diff
@@ -135,4 +139,14 @@ func testFilesCopied(t *testing.T) {
 		assert.Emptyf(t, stdout.String(), "Missing file: %s", fullPath)
 	}
 
+	// Check the SHA of kube.tar.gz downloaded
+	kubeTarPath := ansibleTempDir + "\\" + "kube.tar.gz"
+	command := fmt.Sprintf("certutil -hashfile %s SHA512", kubeTarPath)
+	stdout := new(bytes.Buffer)
+	_, err = client.Run(command, stdout, os.Stderr)
+	require.NoError(t, err, "Error generating SHA512 for %s", kubeTarPath)
+	actualSha := strings.Split(strings.Split(stdout.String(), " ")[3], "\n")[1]
```

Comment:
Please break this up into multiple lines with descriptive variable names at each step and comment what the original string should look like

Reply:
Made the change for cleaner extraction and added an example